uaclient/status.py

```diff
@@ -127,8 +130,13 @@ class UserFacingStatus(enum.Enum):
 STATUS_SERVICE_HEADER = "\nSERVICE"
 STATUS_TMPL = "{name: <14}{entitled: <26}{status}"
 
-MESSAGE_ATTACH_FAILURE_TMPL = """\
-Could not attach machine. Error contacting server {url}"""
+MESSAGE_ATTACH_REQUIRES_TOKEN = """\
+Attach requires a token: sudo ua attach <<TOKEN>>
```

Comment:
The doubling of < and > here looks strange to me. `<TOKEN>` looks more normal to me, and would be in line with the updated usage line (except in terms of case).

@powersj What do you think?

(Regardless of which way we decide here, I think this error message and the usage string should be consistent.)

Reply:
Agreed +1 on single <TOKEN> to be more consistent with other docs as well